src/common/utils/constants.util.ts

```diff
@@ -16,10 +18,19 @@ export const ERROR_MESSAGES = {
   REGISTRATION_END_DATE_INVALID: 'Registration end date must be in the future',
   REGISTRATION_START_DATE_BEFORE_END_DATE:
     'Registration start date must be before registration end date',
-  RECURRENCE_END_DATE_INVALID: 'Recurrence end date must be in the future',
-  RECURRENCE_END_DATE_BEFORE_EVENT_DATE:
+  RECURRENCE_END_DATE_INVALID:
+    'Recurrence end date must be a valid ISO 8601 date string ',
```

Comment:
@Xitija @vaivk369 why we need ISO 8601 in error message, this will create confusion for end user

Reply:
Standard in javascript [](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format)